### PR TITLE
ovn-dbchecker: reset DB only after a fixed time

### PR DESCRIPTION
**Pull request by jcaamano**
When the DB reset was introduced [1], if the DB was unreacheable, it looks like the intention was for the reset to happen after 200 appctl retries * 2 seconds sleep in between * 10+1 retries + 10+1/2*60 (wait time in between ticks) which amounts to ~80 minutes.

Then an extra check was introduced with [2], ensureElectionTimeout, which increased the rate at which the retry counter was increased, reducing that time to ~77 minutes.

Then with [3], the 200 appctl retries * 2 seconds sleep was replaced by a 5s timeout, reducing that time to ~7 minutes.

Moreover, when the DB is not up, is probable that the socket file does not even exist, making appctl to fail inmmediately without timeout, further reduncing that time to 3 minutes.

At that point it takes less time to wipe the DB than the crash-loop back-off time cap of 5 minutes, meaning that we might wipe the DB if the DBs are not up for any other reason that has nothing to do with the DB itself.

This commit proposes to change this to a fixed time value not based on the retries or timeouts but rather absolute that we can tune idependantly.

Basically resetting the DB should be a last resort if we cannot really assess i& fix the raft memebership configuration otherwise. But if we can, even if it is just a glimpse of it for short period, and there is nothing wrong with it, then the DB might not be up for a different reason for which we might have a better chance troubleshooting without restting the DB.

[1] https://github.com/ovn-org/ovn-kubernetes/pull/1930
[2] https://github.com/ovn-org/ovn-kubernetes/pull/2356
[3] https://github.com/ovn-org/ovn-kubernetes/pull/2554

### CODE REVIEW

**Comment by trozet** [comment]
closing this as it is stale and we dont use this for IC
